README.md

```diff
@@ -145,6 +145,9 @@ Enable source-maps via [`devtool`](https://webpack.js.org/configuration/devtool/
 - `minifyIdentifiers` `<Boolean>` - Shorten identifiers
 - `minifySyntax` `<Boolean>` - Use equivalent but shorter syntax
 - `sourcemap` `<Boolean>` (defaults to Webpack `devtool`)- Whether to emit sourcemaps
+- `test` `String|RegExp|Array<String|RegExp>` (`undefined`) - Filter assets for minification
+- `include` `String|RegExp|Array<String|RegExp>` (`undefined`) - Filter assets for inclusion in minification
+- `exclude` `String|RegExp|Array<String|RegExp>` (`undefined`) - Filter assets for exclusion in minification
```

Comment:
Not sure what syntax you'd prefer when there are multiple types for an option. I pulled these straight from the Terser docs, but they don't quite match how you've defined the others.

Reply:
Also, I included all three because that's what Terser does, but `test` and `include` do essentially the same thing. Don't know if we care about keeping to a similar API or not.

Reply:
> Also, I included all three because that's what Terser does, but `test` and `include` do essentially the same thing. Don't know if we care about keeping to a similar API or not.

You're right. Also from [Webpack docs](https://github.com/webpack/webpack.js.org/blob/18e8e38/src/content/configuration/index.mdx#L298):

> test and include have the same behavior, both must be matched


Let's remove the `test` property in favor of `include`.